packages/assets-controllers/src/Standards/ERC20Standard.test.ts

```diff
@@ -12,15 +12,16 @@ const MKR_ADDRESS = '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2';
 describe('ERC20Standard', () => {
   let erc20Standard: ERC20Standard;
   let web3: any;
-  nock.disableNetConnect();
 
   beforeAll(() => {
     web3 = new Web3(MAINNET_PROVIDER);
     erc20Standard = new ERC20Standard(web3);
+    nock.disableNetConnect();
```

Comment:
Some tests hide global configuration changes to Nock in the middle of the file. I've moved these lines to `beforeAll` (and then reverted them in `afterAll`).